### ISSUE TITLE
[Feature Request]: nonparametric factorial ANOVA and pairwise comparisons

### ISSUE THREAD

**Issue by jjxytc** [jasp-stats/jasp-issues#1531]
### Description

conduct nonparametric factorial ANOVA and pairwise comparisons

### Is your feature request related to a problem?

Given that data often violate the assumptions of normality and homoscedasticity, especially with some outliers, small sample sizes and an unbalanced design, it will be very helpful to allow us to conduct nonparametric factorial ANOVA (two-way, three-way, four-way, …, N-way ANOVA) and nonparametric pairwise comparisons when there is a significant main or interaction effect as well as bayesian testing. 

### Describe alternatives that you have considered

In fact, the aligned rank transform (ART) can be used to handle these problems (Elkin et al., 2021; Wobbrock et al., 2011). However, it is not easy for the beginners. If JASP include these functions in the new version, it will become more popular. Elkin, L.A., Kay, M., Higgins, J. and Wobbrock, J.O. (2021). An aligned rank transform procedure for multifactor contrast tests. Proceedings of the ACM Symposium on User Interface Software and Technology (UIST '21). Virtual Event (October 10-14, 2021). New York: ACM Press, pp. 754-768. Wobbrock, J.O., Findlater, L., Gergle, D. and Higgins, J.J. (2011). The aligned rank transform for nonparametric factorial analyses using only ANOVA procedures. Proceedings of the ACM Conference on Human Factors in Computing Systems (CHI '11). Vancouver, British Columbia (May 7-12, 2011). New York: ACM Press, pp. 143-146.

**Comment by boutinb** [comment]
@JohnnyDoorn Is this something you could do?

**Comment by JohnnyDoorn** [comment]
Yes, but it would not be high on my priority list - it's a great feature enhancement (now JASP only does nonparametric one-way ANOVA, including follow-up tests), but I am very busy at the moment..

**Comment by jjxytc** [comment]
@JohnnyDoorn 
Take your time.

**Comment by tomtomme** [comment]
When this gets implemented we should also tackle e.g. missing non-parametric effect sizes:
- [ ] for RM ANOVA https://github.com/jasp-stats/jasp-issues/issues/450
- [ ] for ANOVA https://github.com/jasp-stats/jasp-issues/issues/383
- [ ] move misplaced Dunns post hoc test https://github.com/jasp-stats/jasp-issues/issues/1905
